routes/zooRoute.js

```diff
@@ -0,0 +1,65 @@
+const router = require('express').Router();
+
+const knex = require('knex');
+const knexConfig = require('../knexfile');
+
+const db = knex(knexConfig.development);
+
+router.post('/', async (req, res) => {
+	try {
+		const { id } = await db('zoos').insert(req.body);
+		const zoo = await db('zoos').where({ id }).first();
+		res.status(201).json(zoo);
+	} catch (error) {
+		res.status(500).json({ error: 'post error' });
+	}
+});
+router.get('/', async (req, res) => {
+	try {
+		const zoo = await db('zoos');
+		res.status(200).json(zoo);
+	} catch (error) {
+		res.status(500).json({ error: 'error with get request' });
+	}
+});
+
+router.get('/:id', async (req, res) => {
+	try {
+		const { id } = req.params;
+		db('zoos').where({ id }).then((zoo) => {
+			res.status(200).json(zoo);
```

Comment:
There doesn't seem to be a catch for when the id can't be found; I just get an empty array in postman.